nannyml/base.py

```diff
@@ -177,6 +177,7 @@ def __init__(
         chunk_period: Optional[str] = None,
         chunker: Optional[Chunker] = None,
         timestamp_column_name: Optional[str] = None,
+        calculation_method: Optional[str] = None,
```

Comment:
I would not go so far as to add this on the base class. If I understand it correctly the `calculation_method` parameter only applies to the univariate drift calculator, so I would only introduce it there.